test/unit/utility/range/to_test.cpp

```diff
@@ -0,0 +1,200 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <deque>
+#include <forward_list>
+#include <list>
+#include <seqan3/std/ranges>
+#include <vector>
+
+#include <seqan3/utility/range/to.hpp>
+
+/**
```

Comment:
```suggestion
/*
```

And elsewhere in this file